src/django/api/models.py

```diff
@@ -2143,7 +2141,7 @@ def __str__(self):
 
 
 @transaction.atomic
-def index_facilities(facility_ids=[]):
+def index_facilities(facility_ids=list()):
```

Comment:
I wasn't as clear as I could have been my last comment about providing a default list argument. If we call `list()` here I think we end up in the same problem as using a literal list. A single list is created when Python interprets the function and it shared from then on. The trick to get around this, which is not intuitive, is to use a "callable" as the default value.

```python
def index_facilities(facility_ids=list):
```

Reply:
Got it, thank you!